packages/graph-explorer/src/hooks/useUpdateSchema.ts

```diff
@@ -17,37 +17,28 @@ const useUpdateSchema = () => {
         const currentSchema =
           typeof schema === "function" ? schema(prevSchema) : schema;
 
-        // Preserve vertices counts
-        const vertices = (
-          currentSchema?.vertices ||
-          prevSchema?.vertices ||
-          []
-        ).map(vertex => {
-          const prevVertex = prevSchema?.vertices.find(
-            v => v.type === vertex.type
-          );
+        // Vertices counts
+        const vertices = (currentSchema?.vertices || []).map(vertex => {
```

Comment:
General question, what was the purpose of the previous schema preservation? Would it affect anything else with the removal?

Reply:
The changes won't significantly impact anything besides providing you with the most current and up-to-date numbers, instead of catching them from the previous synchronization payload.

Reply:
Ok, as long as it wasn't for any performance reasons we should consider.